roles/debian-hardening/vars/main.yml

```diff
@@ -7,11 +7,15 @@ kernel_params:
  - init_on_free=1
  - slab_nomerge
  - pti=on
- - slub_debug=ZF
+ - slub_debug=FZP
```

Comment:
Are you sure we need to enable slub poisoning?
According https://kernsec.org/wiki/index.php/Kernel_Self_Protection_Project/Recommended_Settings slug poisoning is not required if init_on_alloc=1
init_on_free=1 is enabled.